templates/login_missing_username.html

```diff
@@ -0,0 +1,18 @@
+<html lang="en">
+  <head>
+      <link href={{url_for('static', filename='style.css')}} rel="stylesheet">
+  </head>
+  <body style="background-color:powderblue;">
+    <h1>Login Error</h1>
+      <h2>The given username is not associated with any registered account. Please provide a registered username.</h2>
```

Comment:
Optional: ```login_missing_username.html```, ```login.html``` and ```incorrect_password.html``` could be merged into one file and just have the "Login Error" as a parameter. ```{% if }``` can be used to check if an error exists and adjust the page as necessary.

Same thing for ```register.htm```l and the corresponding error templates.

Reply:
I also like this idea alot since it simplifies the codebase quite a bit.  Changes are pushed now